src/main/scala/interplay/PlayReleaseBase.scala

```diff
@@ -25,8 +25,6 @@ object PlayReleaseBase extends AutoPlugin {
     // Release settings
     releasePublishArtifactsAction := PgpKeys.publishSigned.value,
     releaseTagName := (version in ThisBuild).value,
-    playCrossReleasePlugins := true,
-    releaseCrossBuild := (playCrossBuildRootProject in ThisBuild).?.value.exists(identity),
```

Comment:
Should you re-instate this too?  `releaseCrossBuild` is a key in sbt-release, so the intent here is to pre-configure it.  Dropping this will change the behaviour of `release`.

Reply:
It is unnecessary and redundant. We already do `+test` and `+publishSigned`.